File: src/main/java/com/kamesuta/power/proofvoteban/ProofVoteBan.java
package com.kamesuta.power.proofvoteban;

import dev.kotx.flylib.FlyLib;
import org.bukkit.plugin.java.JavaPlugin;

public final class ProofVoteBan extends JavaPlugin {

    @Override
    public void onEnable() {
        FlyLib.create(this, builder -> {
            builder.command(new VoteBAN());
        });

    }

    @Override
    public void onDisable() {
    }
}

File: src/main/java/com/kamesuta/power/proofvoteban/ProofVoteBan.kt
package com.kamesuta.power.proofvoteban

import dev.kotx.flylib.flyLib
import org.bukkit.plugin.java.JavaPlugin

class ProofVoteBan : JavaPlugin() {

    override fun onEnable() {
        flyLib {
            command(VoteBanCommand(this@ProofVoteBan))
        }
    }

    override fun onDisable() {}
}